src/test/java/org/zipcodewilmington/lab1/ArrayUtilities.java

```diff
@@ -0,0 +1,52 @@
+package org.zipcodewilmington.lab1;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+public class ArrayUtilities {
+
+    public static int[] moveLastToFirst(int[] values) {
+        {
+            if (values.length < 2)
+                return values;
+            int last = values[values.length - 1];
+
+            for (int i = values.length - 1; i > 0; i--) {
+                values[i] = values[i - 1];
+            }
+            values[0] = last;
+        }
+        return null;
+    }
+
+    public static int[] removeDuplicate(int[] values) {
+        int j = 0;
+        int i = 1;
+        if (values.length < 2)
```

Comment:
Again add curly braces around the if statement. Just in case, in the future, when another developer wants to add another line of code in between the if statement, if they forget to add the curly braces, then the program will have a bug. Another reason to have the curly braces is so you can clearly see where the statement begins and ends.